jetstream/core/server_lib.py

```diff
@@ -94,6 +94,10 @@ def run(
 
   jetstream_server = JetStreamServer(driver, server)
   jetstream_server.start(port, credentials)
+
+  jax.profiler.start_server(9999)
```

Comment:
Let's add the profiler server in the server implementation with a flag to control that? https://github.com/google/JetStream/blob/main/jetstream/core/implementations/mock/server.py
Since we only need it during debugging and profiling, and start an extra server may affect JetStream server's performance.